loops_exercises.js

```diff
@@ -0,0 +1,62 @@
+// Question 1:
+
+let num = 50;
+
+while (num > 0) {
+  console.log(num);
+  num--;
+}
+
+//
+
+let num2 = 25;
+
+for (let i = num2; i > 0; i--) {
+  console.log(i);
+}
+
+
+// Question 2:
+
+
+let num3 = 12;
+
+while (0 <= num3) {
+  if (num3 % 2 === 0) {
+    console.log(num3 + ' is even.');
+  } else { console.log(num3 + ' is odd.');
```

Comment:
This looks good but lets try to write the code inside the block on it's own line.